imports/startup/server/index.coffee

```diff
@@ -44,196 +223,45 @@ escape = (text)->
     JSON.stringify(text)
 
 Meteor.methods
```

Comment:
Could you remove the meteor methods and replace the method calls with `HTTP.get` calls to the Restivus API?